lib/plugin-manager.js

```diff
@@ -90,8 +90,10 @@ class PluginManager extends EventEmitter {
       this._loadSourcesFromIndex();
     });
 
-    this.metadata = new Metadata();
-    this.metadata.service.host = configuration.metadataHost || Config.get('metadata:host');
+    this.metadata = new Metadata({
+      host: configuration.metadataHost || Config.get('metadata:host'),
+      interval: configuration.interval || Config.get('metadata:interval')
```

Comment:
You probably want to get `config.metadataInterval` here so you're not accidentally resolving this as the polling interval for the index.